### PR TITLE
Feature author check fix

### PR DESCRIPTION
**Pull request by sulfo**

### CODE REVIEW

File: src/Bonnier/WP/BestAnswer/Forum/Reply.php [hunk 1]
```diff
@@ -90,21 +90,21 @@ private static function redirect_back()
      */
     private static function set_best_answer($reply)
     {
-        $topicId = bbp_get_reply_topic_id($reply);
+        $topic_id = bbp_get_reply_topic_id($reply);
```
Comment (Frozire) [inline src/Bonnier/WP/BestAnswer/Forum/Reply.php]: wtf this

**Review by Frozire — CHANGES_REQUESTED** [review]
123

**Comment by alfhen** [comment]
@Frozire staaaaaapp

**Comment by Frozire** [comment]
UI bug

**Comment by sulfo** [comment]
@alfhen I agree

**Comment by sulfo** [comment]
@alfhen check igen

**Pull request merged**